src/Form/Extension/ChoiceTypeExtension.php

```diff
@@ -31,17 +31,18 @@ public function configureOptions(OptionsResolver $resolver): void
         $resolver->setDefined($optionalOptions);
     }
 
+    /**
+     * @param array<string, mixed> $options
```

Comment:
can you do array shape for `$options`, it's better then `mixed`

Reply:
It is better to leave it like this for this PR and improve further on next PRs